packaging/windows/OpenRA.nsi

```diff
@@ -126,35 +126,41 @@ Section "Game" GAME
 	File /r "${SRCDIR}\mods\modcontent"
 
 	SetOutPath "$INSTDIR"
-	File "${SRCDIR}\RedAlert.exe"
-	File "${SRCDIR}\TiberianDawn.exe"
-	File "${SRCDIR}\Dune2000.exe"
-	File "${SRCDIR}\OpenRA.Game.exe"
-	File "${SRCDIR}\OpenRA.Game.exe.config"
-	File "${SRCDIR}\OpenRA.Utility.exe"
-	File "${SRCDIR}\OpenRA.Server.exe"
-	File "${SRCDIR}\OpenRA.Platforms.Default.dll"
-	File "${SRCDIR}\ICSharpCode.SharpZipLib.dll"
-	File "${SRCDIR}\FuzzyLogicLibrary.dll"
-	File "${SRCDIR}\Open.Nat.dll"
+	File "${SRCDIR}\*.exe"
+	File "${SRCDIR}\*.exe.config"
+	File "${SRCDIR}\*.dll"
+	File "${SRCDIR}\*.deps.json"
```

Comment:
This will need a matching `Delete` statement for uninstall.

Reply:
Maybe the `.runtimeconfig.json` too? I still can't get it working with both sets of json files though.